c7n/resources/aws.py

```diff
@@ -745,7 +745,7 @@ def get_service_region_map(regions, resource_types, provider='aws'):
             normalized_types.append(r)
     resource_service_map = {
         r: clouds[provider].resources.get(r).resource_type.service
-        for r in normalized_types if r != 'account'}
+        for r in normalized_types if r != 'account' and r}
```

Comment:
I think we discussed this in the previous PR.  Was there a time you knew where `r` might not be truthy?

Reply:
oops yea i think this is a leftover bad merge